ehealthid-rp/src/main/resources/i18n_de_DE.properties

```diff
@@ -0,0 +1,17 @@
+lang=de-DE
+title=Anmeldung mit GesundheitsID
+error.login=Einloggen mit GesundheitsID
+error.serverError=Ohh nein! Unerwarteter Serverfehler. Bitte versuchen Sie es erneut.
+error.noProvider=Kein Identit�tsanbieter ausgew�hlt. Bitte zur�ckgehen.
+error.invalidSession=Oops, Sitzung unbekannt oder abgelaufen. Bitte starten Sie erneut.
+error.insecureRedirect=Unsicherer redirect_uri='%s'. Falsch konfigurierter Server, bitte verwenden Sie 'https'.
+error.badRedirect=Ung�ltige redirect_uri='%s'. �bergebener Link ist nicht g�ltig.
+error.untrustedRedirect=Nicht vertrauensw�rdiger redirect_uri=%s. Falsch konfigurierter Server.
+error.unsupportedScope=Bereich '%s' wird nicht unterst�tzt
+error.unsupportedResponseType=Unterst�tzter Antworttyp: '%s'
```

Comment:
AFAIK unterstützten means "to support" so doesn't this mean the opposite of what you mean?